index.js

```diff
@@ -4,15 +4,46 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
+header.style.color = "red"
 
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+const profile = document.querySelector("#profile")
+const name = profile.children[1];
+name.innerHTML = traveler.name;
```

Comment:
Since you are not putting any html inside of the nickName tag, just text, you can use  .innerText